src/bandmap.c

```diff
@@ -802,14 +812,39 @@ void bandmap_show() {
 
     /* show highlighted frequency marker or spot on QRG if rig control
      * is active */
+    //syslog(LOG_DEBUG, "bandmap_show() called");
     if (trx_control != 0) {
 	move (bm_y, bm_x);
 	attrset(COLOR_PAIR(C_HEADER) | A_STANDOUT);
 	if (!on_qrg) {
 	    printw ("%7.1f   %s", centerfrequency,  "============");
+	    if (bmautograb != 0 && autograbbed == 1 && cqmode == S_P) {
+		hiscall[0] = '\0';
+		showinfo( getctydata( hiscall ) );
+		searchlog( hiscall );
+		autograbbed = 0;
+	    }
+	    // clear hiscall if no qrg
+	    if (bmautoadd == 1 && bmadded_spot == 1 && strlen(hiscall) > 0) {
```

Comment:
I assume that the code below should handle the case that we tuned frequency after the auto_add. Then the added call will be cleared?

I think it is not the right place to do that.  From a logical point of view it should be done in that section which added the call to the bandmap - that would be callinput. What do you think?